rules/libyang3-py3.mk

```diff
@@ -0,0 +1,14 @@
+# libyang python3 wheel
+
+LIBYANG3_PY3_VERSION = 3.0.3
+LIBYANG3_PY3_ABI := cp$(shell $(SETARCH) python3 -c "import sysconfig; print(sysconfig.get_python_version().replace('.',''))")
+LIBYANG3_PY3_PLAT := $(shell $(SETARCH) python3 -c "import sysconfig; print(sysconfig.get_platform().replace('.','_').replace('-','_'))")
+LIBYANG3_PY3 := libyang-$(LIBYANG3_PY3_VERSION)-$(LIBYANG3_PY3_ABI)-$(LIBYANG3_PY3_ABI)-$(LIBYANG3_PY3_PLAT).whl
```

Comment:
Why is LIBYANG3_PY3_PLAT necessary for Python wheels?
For instance, with swsssdk-2.0.1-py3-none-any.whl, we don't specify a platform for Python wheels.

Reply:
because this is a cffi python module meaning there is a platform-specific .so generated since it needs bindings to a C library.  The alternative would be to generate a platform-specific .deb with the bindings.  The wheel was just the easiest way ... but considering the armhf bindings it may be better to try to generate a .deb.

I can look into that conversion.  I looked at using the python setuputils stdeb stuff but I don't think that will work.